releases/net471/KnownIssues/470006-GC Crashes with high rate of large object allocation.md

```diff
@@ -0,0 +1,36 @@
+# Rare crashes can occur when background GC is enabled in applications which allocate and free large objects at a high rate
+
+## Symptoms
+
+When background GC is enabled, an application running against .NET Framework 4.7.1 experiences
+intermittent crashes which did not occur when running against other .NET Framework versions.
+
+## Cause
+
+Changes to the GC in .NET Framework 4.7.1 caused this problem.
+
+Specifically, .NET Framework 4.7.1 contains changes which improve background GC performance.
+These changes unintentionally introduced cases where a rare combination of factors can cause
+the background GC to reclaim a large object while the program is still using it.
+
+## Impact
+
+This problem impacts applications which run with the background GC enabled and which also allocate
+and free large objects at a high rate. As described in [this article](https://docs.microsoft.com/en-us/dotnet/standard/garbage-collection/fundamentals#the-managed-heap),
```

Comment:
Rather than a reference to "this article", which tends to be a bit mysterious not really help the reader decide whether they want to follow the link, it 's best to include the title: \[Fundamentals of Garbage Collection]